local-volume/README.md

```diff
@@ -98,30 +98,81 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 
 ### Step 2: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Generating Provisioner's daemonset spec and config map by using Helm chart 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option allows a user to generate Provisioner's daemon set and confgmap using **helm** templating based on predefined values,
+manual modify generated Provisioner's spec and instantiate Provisioner's daemon set and config map with **kubectl**.
+**helm template** uses 3 pieces of information:
+1. Provisioner's chart template located at helm/provisioner/templates/provisioner.yaml
+2. Provisioner's default values.yaml which contains variables used for rendering a template.
+3. (Optional) User's customized values.yaml as a part of helm template command. User's provided
+   values will override default values of Provisioner's values.yaml.
+
+Below is the default values.yaml, user should not remove variables from this file but can change any values of these variables. 
+The only exception is the list of storage classes and respective mounting points. In general there should be at least one storage class
```

Comment:
mount points.  At least one storage class config is required.

Reply:
done